.github/ISSUE_TEMPLATE/--bug-report.yml

```diff
@@ -0,0 +1,46 @@
+name: "🐛Bug report"
+description: "Create a bug report about GDevelop or the game engine"
+body:
+  - type: textarea
+    attributes:
+      label: Describe the bug
+      description: |
+        A clear and concise description of what the bug is.
+        Please double check that the bug is not already reported in the issues list.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps to reproduce
+      description: |
+        * Please include a link to a game if possible!
+        * If applicable, add screenshots to help explain your problem.
+      placeholder: |
+        1. Go to '...'
+        2. Click on '...'
+        3. Scroll down to '...'
+        4. See error
+    validations:
+      required: true
+  - type: dropdown
```

Comment:
I would split this question in 2!
One for the web/desktop version, and one for the actual GDevelop version.
The first one can be a dropdown with web/desktop like you did.
The second one has to be a free text field, with a placeholder like: `5.1.159? 5.1.160?`

Reply:
It seems that two types cannot be used with the same label.:cry:
So I divided it with the image below. And I changed the selection list item to [Desktop, Web]. What do you think?
![image](https://user-images.githubusercontent.com/88178296/233580381-7d0117d8-bc70-4db1-a654-b1295fe71b4b.png)
![image](https://user-images.githubusercontent.com/88178296/233580882-99dfb2f2-5e05-4038-b274-e537ad1ae78a.png)

Reply:
Done!